### ISSUE TITLE
Drawer onClose, tooltip remains unless re-hover

### ISSUE THREAD

**Issue by TheDarkX** [mantinedev/mantine#1146]
### What package has an issue

@mantine/core

### Describe the bug

When drawer closed, the tooltip remains visible beside a button / actionIcon.

Users might forced to 
1) re-hover the button / actionIcon
2) perform a click somewhere else that are not related 

to close the tooltip.

![image](https://user-images.githubusercontent.com/7087449/161126779-47ab3c8e-9996-4db5-86e3-57539ee9a11c.png)
![image](https://user-images.githubusercontent.com/7087449/161126836-13ecbe7b-43ca-45f6-a05f-ebfab3c63e17.png)

### In which browser did the problem occur

Firefox 98.0.2 , Chrome 100.0.4896.60  (currently latest version)

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/currying-glitter-n6oltq

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

### Possible fix

Maybe something related to focus when drawer closed?
Will research regarding this in leisure time.

**Comment by JuzioMiecio520** [comment]
With the CodeSandbox link you have provided, your issue seems to be fixed, but I just had a similar problem.

This works just fine:
```jsx
<Tooltip label="hi">
  <Button>
    <AiOutlineArrowRight />
  </Button>
</Tooltip>
```
But as soon as you add the `disabled` property to the button, the tooltip remains open, even after you re-hover:
```jsx
<Tooltip label="hi">
  <Button disabled={true}>
    <AiOutlineArrowRight />
  </Button>
</Tooltip>
```

Any ideas?

**Comment by rtivital** [comment]
Tooltip does not work with disabled button because react does not handle mouseleave events – https://github.com/facebook/react/issues/18753

**Comment by JuzioMiecio520** [comment]
Oh, thanks!

**Comment by dukesx** [comment]
> ### What package has an issue
> @mantine/core
> 
> ### Describe the bug
> When drawer closed, the tooltip remains visible beside a button / actionIcon.
> 
> Users might forced to
> 
> 1. re-hover the button / actionIcon
> 2. perform a click somewhere else that are not related
> 
> to close the tooltip.
> 
> ![image](https://user-images.githubusercontent.com/7087449/161126779-47ab3c8e-9996-4db5-86e3-57539ee9a11c.png) ![image](https://user-images.githubusercontent.com/7087449/161126836-13ecbe7b-43ca-45f6-a05f-ebfab3c63e17.png)
> 
> ### In which browser did the problem occur
> Firefox 98.0.2 , Chrome 100.0.4896.60 (currently latest version)
> 
> ### If possible, please include a link to a codesandbox with the reproduced problem
> https://codesandbox.io/s/currying-glitter-n6oltq
> 
> ### Do you know how to fix the issue
> No
> 
> ### Are you willing to participate in fixing this issue and create a pull request with the fix
> No
> 
> ### Possible fix
> Maybe something related to focus when drawer closed? Will research regarding this in leisure time.

Same issue happening with me too. Unless you rehover over the tooltip, it wont go. In my case , lets suppose you open a modal with a button having a tooltip. When you close the modal , the tooltip over the button remains. It wont go. 

Note: My button is NOT disabled.

**Comment by rtivital** [comment]
Will be fixed in 5.0.0

**Comment by rtivital** [comment]
Fixed in 5.0 – https://codesandbox.io/s/shy-mountain-0f9jp0?file=/src/App.js
